pkg/query/columnquery.go

```diff
@@ -367,8 +378,7 @@ func filterRecord(
 	pool memory.Allocator,
 	rec arrow.Record,
 	filterQueryBytes []byte,
-	binariesToExclude [][]byte,
-	showInterpretedOnly bool,
+	frameFilter []string,
```

Comment:
I'd make this more specific, make this `binaryFrameFilter` and make it a set, we don't want to do the `bytes.Contains` anymore, that was just because we were guessing the binary names, but now we can do exact matches, for which a set-inclusion test is much better.

Reply:
can we still do a set inclusion for an exact match since the binary name we'll be getting from the frontend is typically something like `libpython3.11.so.1.0` but the mapping file itself is more like `/usr/lib/libpython3.11.so.1.0` or `kubelet` vs `/home/kubernetes/bin/kubelet`?